src/PrestaShopBundle/Resources/config/services/adapter/feature.yml

```diff
@@ -60,6 +60,20 @@ services:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Feature\Command\BulkDeleteFeatureCommand
 
+  PrestaShop\PrestaShop\Adapter\Feature\CommandHandler\DeleteFeatureValueHandler:
+    autowire: true
+    public: false
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\Feature\Command\DeleteFeatureValueCommand
+
+  PrestaShop\PrestaShop\Adapter\Feature\CommandHandler\BulkDeleteFeatureValueHandler:
+    autowire: true
+    public: false
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\Feature\Command\BulkDeleteFeatureValueCommand
+
```

Comment:
Depending on which is merged first, there will be changes needed ont this PR or thi one https://github.com/PrestaShop/PrestaShop/pull/32787